### PR TITLE
ZOOport - FAQ

### PR DESCRIPTION
**Pull request by dguzenko**

### CODE REVIEW

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **Is an option "Import from installed software : Joomla / K2" supported?** - CSV import is supported, if the software allows to export the data in CSV. JSON import supports only JSON files generated by ZOOport, due to the specific format.
```
Comment (ameli90) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: this block is removed here https://github.com/JOOlanders/ZOOport/commit/403d26af7074dc7ab55e259557b1ef9dbd32da80 as is not supported

**Comment by ameli90** [comment]
It would be nice to add the next question:

- Why ZOOport wont import or import not correct symbols? - The file to import should be UTF-8 encoded.

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md (hunk 1 above)
Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: Please give more details.
And I can't open the link.

**Comment by dguzenko** [comment]
Thank you! Done

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md (hunk 1 above)
Comment (ameli90) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: We can remove the question about "Import from installed software : Joomla / K2" option as this block is removed now on import page

Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: But, in general, users can be interested in this feature (even they don't see this block), as they know that standard import has it.

Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: Vika, but is the answer correct or not? :)

Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: We have discussed and agree to remove this question.

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **What should be the date format in a CSV file?** - The correct date format is YYYY-MM-DD HH:MM:SS. If a CSV file won't import, please make sure you are using the correct date format.
+
+* **Is it possible to import multiple Select element values?** - Yes, it is possible with CSV import. All possible options should be specified in the Select element settings. In the CSV file should be a separate column for each option, which is equal to the options' Values, not Names.
```
Comment (miljan-aleksic) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: This answer is unclear, difficult to understand. Provide an example of how should be formatted the data.

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **What should be the date format in a CSV file?** - The correct date format is YYYY-MM-DD HH:MM:SS. If a CSV file won't import, please make sure you are using the correct date format.
+
+* **Is it possible to import multiple Select element values?** - Yes, it is possible with CSV import. All possible options should be specified in the Select element settings. In the CSV file should be a separate column for each option, which is equal to the options' Values, not Names. 
+
+* **How to import items as Pulished / Searchable / Frontpage?** - In the CSV file add columns "State" / "Searchable" / "Frontpage" with the values "1" or "true" - if "Yes", and "0" or "false" - if "No". Then during Assign Element step assign them to the Core Attributes fields.
+
+* **Why ZOOport wont import or import not correct symbols?* - The file to import should be UTF-8 encoded.
```
Comment (miljan-aleksic) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: The question is missing the enclosing `*`

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md (hunk 2 above)
Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: Changed the answer and added images. But Vika should help me with uploading images.

File: 02.extensions/zooport/02.troubleshooting/02.faq/docs.md (hunk 3 above)
Comment (dguzenko) [inline 02.extensions/zooport/02.troubleshooting/02.faq/docs.md]: Fixed

**Comment by miljan-aleksic** [comment]
Don't use images, just code examples as this one here:

```
Some code example
```

**Comment by dguzenko** [comment]
But what is not clear? How the CSV with separate columns looks like or how to use Values, not Names? What case requires the example?

**Comment by miljan-aleksic** [comment]
If somebody wants to create an CSV file from scratch requires an example, real one. Doesn't matter now, leave it as it is without images.

**Comment by dguzenko** [comment]
Images removed

**Pull request merged**